src/backend/columnar/sql/udfs/columnar_metapage/10.1-1.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE FUNCTION columnar.columnar_metapage(
+    rel regclass,
+    version_major OUT int4,
+    version_minor OUT int4,
```

Comment:
Sounds good. I just put it there because we were already adding upgrade_columnar_metapage, but if I put that in the citus_internal schema then I'll move this one back to test helpers.